tests/rest_api/test_projects.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (C) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from http import HTTPStatus
+import pytest
+
+from .utils.config import get_method, post_method, patch_method
+
+
+class TestGetProjects:
+    def _find_by_owner(self, projects, users, is_project_staff, is_org_member, is_owner, has_membership):
+        for p in projects:
+            for u in users:
+                if is_owner and not has_membership:
+                    if is_project_staff(u['id'], p['id']):
+                        return u['username'], p['id']
+                if has_membership and not is_owner:
+                    if p['organization'] is not None:
+                        if is_org_member(u['id'], p['organization']):
+                            return u['username'], p['id']
+                if is_owner and has_membership:
+                    if is_project_staff(u['id'], p['id']):
+                        if is_org_member(u['id'], p['organization']):
+                            return u['username'], p['id']
+                if not is_owner and not has_membership:
+                    if not is_project_staff(u['id'], p['id']) and not is_org_member(u['id'], p['organization']):
+                        return u['username'], p['id']
+
+    def _test_response_200(self, username, project_id):
+        response = get_method(username, f'projects/{project_id}')
```

Comment:
```suggestion
    def _test_response_200(self, username, project_id, **kwargs):
        response = get_method(username, f'projects/{project_id}', **kwargs)
```
This test class may require additional arguments (for example: org_id)